CMakeLists.txt

```diff
@@ -0,0 +1,234 @@
+cmake_minimum_required (VERSION 2.8.11)
+project (libpd C)
+
+option(BUILD_SHARED_LIBS "Build libpd as a shared library instead of static" ON)
+option(pd_utils "Compile utilities" ON)
+option(pd_extra "Compile extras" ON)
+option(pd_multi "Compile with multiple instance support" ON)
+option(pd_locale "Set the LC_NUMERIC number format to the default C locale" ON)
+option(pd_build_c_examples "Builds C API example programs" OFF)
+option(COVERALLS "Build with coveralls" OFF)
+
+if(WIN32)
+    set(CMAKE_C_FLAGS  "${CMAKE_C_FLAGS} -DWINVER=0x502 -DWIN32 -D_WIN32")
+endif()
+if (MSVC)
+    set(CMAKE_C_FLAGS  "${CMAKE_C_FLAGS} -DHAVE_STRUCT_TIMESPEC")
+    add_definitions("/D_CRT_SECURE_NO_WARNINGS /wd4091 /wd4996")
+    if(${CMAKE_SIZEOF_VOID_P} EQUAL 8)
+        set(CMAKE_C_FLAGS  "${CMAKE_C_FLAGS} -DPD_LONGINTTYPE=\"long long\"")
+    endif()
+else()
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast")
+    set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS} -ffast-math -funroll-loops -fomit-frame-pointer -O3")
+    set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS} -g -O0")
+endif()
+add_definitions(-DPD=1 -DUSEAPI_DUMMY=1)
+
+if(COVERALLS)
```

Comment:
I guess it's not necessary for the original libpd repository

Reply:
Sure. I will remove it. I tried to keep the functionality of your script as well.